.github/workflows/ci-tests-fabric.yml

```
@@ -108,6 +108,7 @@ jobs:
         FREEZE_REQUIREMENTS: 1
       run: |
         pip install -e .[test] "pytest-timeout" --upgrade --find-links ${TORCH_URL}
+        pip install -r ./requirements/fabric/strategies.txt
```

```suggestion
        pip install -e .[test,strategies] "pytest-timeout" --upgrade --find-links ${TORCH_URL}
```
eventually resolving version collisions :otter:

---

I tried this but it didn't work.